stories/62-Progress.stories.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { MLProgress } from 'marklogic-ui-library'
+import { number, withKnobs, radios } from '@storybook/addon-knobs'
+
+export default {
+  component: 'MLProgress',
```

Comment:
`component` is given as the component, rather than a string name, in another PR. should this be changed?